qiita_pet/templates/study_ajax/base_info.html

```diff
@@ -114,6 +142,10 @@
       <b>Shared With:</b> <span id="shared_html_{{study_info['study_id']}}"></span><br/>
       <b>Samples:</b> {{study_info['num_samples']}}<br />
       <b>EBI:</b> {% raw ebi_info %}<br />
+      <b>Public raw data:</b> {{study_info['public_raw_download']}}
```

Comment:
I feel like this controller is a bit awkward. What about making the text be "Allow Qiita users to download raw data files:", and making the button+true/false a checkbox that is preset to checked/unchecked depending on the status.